MabiCommerce/data/db/modifiers.json

```diff
@@ -21,50 +21,55 @@
 	},
 	{
 		"Id" : 3,
+		"Name": "(Seasonal)Gavin's Special Clothes",
+		"SpeedBonus": 0.6 
+	},
+	{
+		"Id" : 4,
```

Comment:
Did the IDs of these modifiers actually change? I can't remember offhand if the IDs are tied to actual mabi systems or if I just generated them arbitrarily.

Reply:
With the new mounts, the Dog has a buff in "snow fields" aka Vales, while the Camel has a buff in desert(Filia, not Muyu) AND a debuff in snow.

Reply:
👍